src/buildercore/terraform.py

```diff
@@ -857,16 +860,16 @@ def populate_data(self, type, name, block=None):
         self.data[type][name] = block
 
     def populate_local(self, name, value):
-        self.locals[name] = value
+        self.local[name] = value
 
     def to_dict(self):
         result = {}
         if self.resource:
             result['resource'] = self.resource
         if self.data:
             result['data'] = self.data
-        if self.locals:
-            result['locals'] = self.locals
+        if self.local:
+            result['locals'] = self.local
```

Comment:
I think if this needs to end up in `locals` then `local` could be confusing or too subtle (easy to miss the `s` in the end). Maybe better to name the parameter and instance variable `_locals`? (at least that would be my preference I think)

Reply:
I think the standard Python idiom to avoid clashes would be `locals_`; https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles has `class_`